examples/recipe-sourcing-contentful/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "gatsby-starter-default",
```

Comment:
This package.json file should be customized a bit for the example.

Reply:
Done in https://github.com/gatsbyjs/gatsby/pull/17318/commits/6bf620110f66fc4a9f9a982c7c38f32d166316b2